pkg/innerring/processors/container/process_container.go

```diff
@@ -66,7 +66,7 @@ func (cp *Processor) checkPutContainer(e *containerEvent.Put) error {
 
 func (cp *Processor) approvePutContainer(e *containerEvent.Put) {
 	// FIXME: here we should bind key to owner if needed
```

Comment:
This FIXME comment is not relevant anymore.

Reply:
Thx, fixed